bin/server.py

```diff
@@ -57,7 +57,10 @@ def __init__(self, application, request, **kwargs):
 
     def destroy(self):
         if self._container_name:
-            self._client.stop(self._container_name)
+            try:
+                self._client.stop(self._container_name)
+            except docker.errors.NotFound:
+                pass
```

Comment:
```python
            # When the user presses Ctrl-c, QEMU and its container exit. Since
            # the container runs with the --rm option, after stopping it's
            # removed.
            self.logger.info('The container {} '
                             'does not exist'.format(self._container_name))
```